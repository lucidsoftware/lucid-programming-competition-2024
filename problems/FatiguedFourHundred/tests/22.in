version https://git-lfs.github.com/spec/v1
oid sha256:c84f0faeaed6c393542659e82ca82ec669af1b09f1e93721d6ad9037d3f6af71
size 482469
