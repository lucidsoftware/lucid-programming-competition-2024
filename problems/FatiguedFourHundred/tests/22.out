version https://git-lfs.github.com/spec/v1
oid sha256:aa353664dd9df1c3f7e904e3bbfcfa4a10f9f2e7c03196df061d1fb5cf8e8146
size 356551
